solutions/devsprint-cecilia-saraiva-1/GitHubApp/Screens/List/ListViewController.swift

```diff
@@ -34,14 +34,19 @@ final class ListViewController: UIViewController {
 
     override func viewDidAppear(_ animated: Bool) {
 
-        service.fetchList { repositories in
-
-            DispatchQueue.main.async {
-
-                self.listView.updateView(with: repositories)
+        service.fetchList(username: "lysonjeada") { result in
+            
+            do {
+                let repositories = try result.get()
+                print(repositories)
+                DispatchQueue.main.async {
+                    self.listView.updateView(with: repositories)
+                }
+            }
+            catch {
```

Comment:
[suggestion] since we are covering errors, maybe here we can print in case there's an error.